libraries/protocol/include/graphene/protocol/htlc.hpp

```diff
@@ -120,8 +120,10 @@ namespace graphene { namespace protocol {
          struct fee_parameters_type {};
 
          htlc_redeemed_operation() {}
-         htlc_redeemed_operation( htlc_id_type htlc_id, account_id_type from, account_id_type to, account_id_type redeemer, asset amount ) :
-               htlc_id(htlc_id), from(from), to(to), redeemer(redeemer), amount(amount) {}
+         htlc_redeemed_operation( htlc_id_type htlc_id, account_id_type from, account_id_type to, account_id_type redeemer, asset amount,
```

Comment:
Please wrap this line (max = 117 chars).

Reply:
Done.